libs/configuration/src/lib/configuration/configuration.component.ts

```diff
@@ -1,11 +1,95 @@
-import { CommonModule } from '@angular/common';
-import { Component } from '@angular/core';
+import { AsyncPipe, NgIf } from '@angular/common';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { FormBuilder, ReactiveFormsModule } from '@angular/forms';
+import { Project } from '@angular-cli-gui/shared/data';
+import { JsonObject } from '@angular-devkit/core/src/json/utils';
+import { FormlyFieldConfig, FormlyModule } from '@ngx-formly/core';
+import { FormlyJsonschema } from '@ngx-formly/core/json-schema';
+import { FormlyMaterialModule } from '@ngx-formly/material';
+import {
+  combineLatest,
+  forkJoin,
+  map,
+  Observable,
+  Subject,
+  switchMap,
+  tap,
+} from 'rxjs';
+
+import { WorkspaceSettingsService } from '../services';
 
 @Component({
   selector: 'cli-configuration',
   standalone: true,
-  imports: [CommonModule],
+  imports: [
+    NgIf,
+    AsyncPipe,
+    FormlyModule,
+    FormlyMaterialModule,
+    ReactiveFormsModule,
+  ],
   templateUrl: './configuration.component.html',
   styleUrls: ['./configuration.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class ConfigurationComponent {}
+export class ConfigurationComponent implements OnInit {
```

Comment:
if observables are being subscribed in more than once place you should also add `shareReplay({ bufferSize: 1, refCount: true })` operator at the end, in order to avoid multiple subscriptions to the source